go.mod

```diff
@@ -130,7 +130,7 @@ require (
 )
 
 replace (
-	github.com/CosmWasm/wasmd v0.28.0 => github.com/neutron-org/wasmd v0.28.0-keep-ibc-events
+	github.com/CosmWasm/wasmd v0.28.0 => github.com/neutron-org/wasmd v0.28.1-0.20221130133614-9661821bf976
```

Comment:
We should keep in mind to replace it with `neutron` branch once PR to wasmd will be merged